projects/framework-maya/release_notes.md

```diff
@@ -1,5 +1,11 @@
 # ftrack Framework Maya integration release Notes
 
+## upcoming
+
+
+* [new] Scene loader tool added. Reference and import .ma and .mb scenes supported.
```

Comment:
Perhaps align release note somewhat with Nuke - mention that it is driven from Studio (actions) so user's don't expect a new loader tool to be present in Maya menu.